scripts/dts/gen_defines.py

```diff
@@ -1031,15 +1031,14 @@ def out_comment(s: str, blank_before=True) -> None:
         for line in s.splitlines():
             # Avoid an extra space after '*' for empty lines. They turn red in
             # Vim if space error checking is on, which is annoying.
-            res.append(" *" if not line.strip() else " * " + line)
+            res.append(" *" if not line.strip() else f" * {line}")
```

Comment:
```suggestion
            res.append(f" * {line}".rstrip())
```

Reply:
It seems to break the original functionality :(
e.g.
```
>>> line = 'aa '
>>> f" * {line}".rstrip()
' * aa'
>>> " *" if not line.strip() else f" * {line}"
' * aa '
```

Reply:
We never want trailing spaces, so I don't understand the argument?

Reply:
I checked and there won't be a boundary case of trailing spaces, I will change that :)